gnd/build.gradle

```diff
@@ -152,14 +152,27 @@ android {
                 // Specify the packages which NullAway should analyze.
                 option("NullAway:AnnotatedPackages", "com.google.android.gnd")
 
+                // Treat generated classes from Hilt and SafeArgs as unannotated.
+                option("NullAway:TreatGeneratedAsUnannotated", true)
+
                 // Ignore generated classes from databinding, as well as Room and Firebase classes.
                 option("NullAway:UnannotatedSubPackages",
                     "com.google.android.gnd.databinding," +
                         "com.google.android.gnd.persistence.local.room," +
                         "com.google.android.gnd.persistence.remote.firestore," +
                         "com.google.android.gnd.persistence.remote.firestore.schema")
 
-                excludedPaths = ".*/build/generated/.*"
+                // TODO(#718): Make this list dynamic somehow.
+                option("NullAway:ExcludedClasses",
+                    "com.google.android.gnd.ui.observationdetails.ObservationDetailsFragmentArgs," +
+                        "com.google.android.gnd.ui.observationdetails.ObservationDetailsFragmentDirections," +
+                        "com.google.android.gnd.ui.editobservation.EditObservationFragmentArgs," +
+                        "com.google.android.gnd.ui.home.HomeScreenFragmentArgs," +
+                        "com.google.android.gnd.ui.home.HomeScreenFragmentDirections," +
+                        "com.google.android.gnd.ui.offlinebasemap.viewer.OfflineBaseMapsFragmentArgs," +
+                        "com.google.android.gnd.ui.offlinebasemap.OfflineBaseMapsFragmentDirections," +
+                        "com.google.android.gnd.ui.offlinebasemap.viewer.OfflineBaseMapViewerFragmentArgs"
+                )
```

Comment:
Please revert these changes. It probably happened during merge conflicts.

Reply:
reverted